packages/ember-runtime/lib/system/core_object.js

```diff
@@ -418,7 +435,21 @@ CoreObject.PrototypeMixin = Mixin.create({
     @default false
     @public
   */
-  isDestroying: false,
+  isDestroying: descriptor({
+    get() {
+      let m = meta(this);
+      return (m._flags & SOURCE_DESTROYING) !== 0;
```

Comment:
Same, same ... not sure it's worth it to take a hard line against bitmasks that aren't even publicly exposed.